packages/ai-bot/helpers.ts

```diff
@@ -42,7 +42,8 @@ You may make multiple function calls, all calls are gated by the user so multipl
 If a user asks you about things in the world, use your existing knowledge to help them. Only if necessary, add a *small* caveat at the end of your message to explain that you do not have live external data. \
 \
 If you need access to the cards the user can see, you can ask them to attach the cards. \
-If you encounter JSON structures, please enclose them within backticks to ensure they are displayed stylishly in Markdown.';
+If you encounter JSON structures, please enclose them within backticks to ensure they are displayed stylishly in Markdown. \
+Always provide a content text explanation before calling a function';
```

Comment:
@IanCal does this need to be phrased differently to help it keep the context text and the function in the same response?

Reply:
It'll never return more than one response, I would be tempted to leave this until we see lots of calls we want an explanation with coming without them, and ideally have evals to see the difference it makes. It may be different for different models.